.chloggen/add_host_cpuinfo.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: processor/resourcedetection
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Added support for host's cpuinfo attributes. This information is retrieved from /proc/cpuinfo.
```

Comment:
Wondering if we should remove the `/proc/cpuinfo` bit. On Windows builds for example this would not be the case, and this wording may lead people to think that this is a Linux-only feature.

Reply:
Hmm, I think we can generalize yes. However in Windows not all of the attributes will be populated by the lib: https://github.com/shirou/gopsutil/blob/master/cpu/cpu_windows.go#L113-L122

Reply:
I guess we can list the ones that are Linux only if you think that makes sense as the `subtext`